cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -616,6 +619,276 @@ async def _get_trustworthiness_score_async(
+            Models currently supported include "gpt-3.5-turbo-16k", "gpt-4", "gpt-4o", "claude-3-haiku".
+
+        quality_preset (TLMQualityPreset, default = "medium"): preset configuration to control the quality of TLM trustworthiness scores vs. runtimes/costs.
+            Note that this preset only applies to the model computing the trustworthiness score.
+            Supported options include: "medium", "low".
+
+        options (TLMOptions, optional): a typed dict of advanced configuration options.
+            Note that most of these options only apply to the model computing the trustworthiness score, except for "max_tokens", which applies to both models.
+            Available options (keys in this dict) include "model", "max_tokens", "num_consistency_samples", "use_self_reflection".
+            For more details about the options, see the documentation for [TLMOptions](#class-tlmoptions).
+
+        timeout (float, optional): timeout (in seconds) to apply to each TLM prompt.
+
+        verbose (bool, optional): whether to print outputs during execution, i.e., whether to show a progress bar when TLM is prompted with batches of data.
+    """
+
+    def __init__(
+        self,
+        api_key: str,
+        response_model: str,
+        quality_preset: TLMQualityPreset,
+        *,
+        options: Optional[TLMOptions] = None,
+        timeout: Optional[float] = None,
+        verbose: Optional[bool] = None,
+    ) -> None:
+        """
+        Use `Studio.TLMHybrid()` instead of this method to initialize a TLMHybrid object.
+        lazydocs: ignore
+        """
+        self._api_key = api_key
+        self._response_model = response_model
+
+        if quality_preset in {"high", "best"}:
+            raise ValidationError(
+                f"Invalid quality preset: {quality_preset}. TLMHybrid only supports 'low' and 'medium' presets."
+            )
+        self._score_quality_preset = quality_preset
+
+        if options is not None:
+            validate_tlm_hybrid_score_options(options)
+        self._score_options = options
+        self._prompt_options = cast(
+            TLMOptions, get_tlm_hybrid_prompt_options(self._score_options, self._response_model)
+        )
+
+        self._timeout = timeout  # TODO: are we handling this any differently?
+        self._verbose = verbose
+
+        self.tlm_response = TLM(
+            self._api_key,
+            quality_preset="base",
+            options=self._prompt_options,
+            timeout=self._timeout,
+            verbose=self._verbose,
+        )
+
+        self.tlm_score = TLM(
+            self._api_key,
+            quality_preset=self._score_quality_preset,
+            options=self._score_options,
+            timeout=self._timeout,
+            verbose=self._verbose,
+        )
+
+    def prompt(
+        self,
+        prompt: Union[str, Sequence[str]],
+    ) -> Union[TLMResponse, List[TLMResponse]]:
+        """
```

Comment:
Instead of all this could we just point the reader to the TLM prompt() documentation?